pom.xml

```diff
@@ -947,7 +947,18 @@
           <version>${maven-surefire.version}</version>
           <configuration>
             <includes>
+              <include>org.openmetadata.service.apps.*.java</include>
```

Comment:
why are we not saying `org.openmetadata.service.*`? Not sure I understand why we need all the includes in there if we just want to ensure we are running every test